code B.html

```diff
@@ -43,10 +43,29 @@
     <div class="center2"><b>Code B</b></div>
     <div class="center"><img style="width:250px" src="STEM Pictures/STEM logo1.png" alt="STEM_Logo"></div>
   </div>
-  <!--End of Menu section--)
+  <!--End of Menu section-->
 
 <h3>Our Program</h3>
-<body><strong> Code B is one of many opertunties that allows future programers or games design to build a path for themselves or to even sharpen their skills.  BGHS STEM Alumni founded the program and are working diligently to create informative tutorials on how to design games. You will learn step by step on how to design and code games with informative tutorials lasting 5-15 minutes. Learn what it takes to make a professional or fun game with NASA software engineer Julian Vu and experience the fun process of game design</strong>
+  <style>
+p{
+  border: 7px solid black;
+  margin: 10px
+}
+img{
+border:5px solid black;
+margin:10px
+}
+</style>
+<p><strong> Code B is one of many opertunties that allows future programers or games design to build a path for themselves or to even sharpen their skills.  BGHS STEM Alumni founded the program and are working diligently to create informative tutorials on how to design games. You will learn step by step on how to design and code games with informative tutorials lasting 5-15 minutes. Learn what it takes to make a professional or fun game with NASA software engineer Julian Vu and experience the fun process of game design</strong>
 
+
+</p>
+<h4> <strong> Developers of Code B </h4> </strong>
```

Comment:
make sure to nest the element, and don't alternate them . So something like this: "<h4><strong>...</strong></h4>